build/pmm-client-docker/entrypoint.sh

```diff
@@ -3,6 +3,37 @@
 set -o errexit
 set -o xtrace
 
+wait_for_url() {
+    local URL=$1
+
+    for i in `seq 6` ; do
+        curl -k "${URL}" > /dev/null 2>&1 && result=$? || result=$?
+        if [ $result -eq 0 ] ; then
+            return
+        fi
+        sleep 10
+    done
+
+    echo "Operation timed out" >&2
+    exit 1
+}
+
+wait_for_port() {
+    local HOST=$1
+    local PORT=$2
+
+    for i in `seq 6` ; do
+        nc -z "${HOST}" "${PORT}" > /dev/null 2>&1 && result=$? || result=$?
+        if [ $result -eq 0 ] ; then
+            return
+        fi
+        sleep 10
```

Comment:
I don't like such large waiting steps. How long it takes currently, on average, to pass those checks? Is that less than 10s but just sometimes more? Or it's more close to 50s on average?

Reply:
usualy 10s is enoght

Reply:
reworked to make 1s checks